src/Microdown-Blog/MicSingleSummarizer.class.st

```diff
@@ -2,12 +2,63 @@ Class {
 	#name : 'MicSingleSummarizer',
 	#superclass : 'Object',
 	#instVars : [
-		'maximumWords'
+		'maximumWords',
+		'destinationDirectory',
+		'filePath'
 	],
 	#category : 'Microdown-Blog',
 	#package : 'Microdown-Blog'
 }
 
+{ #category : 'rendering' }
+MicSingleSummarizer >> createHtmlFile: aMicRoot [
+
+	| a q date header |
+	a := (MicHTMLVisitor new visit: aMicRoot) at: 1.
+
```

Comment:
we can add on microroot something like 
```
date
     self children first date
```

```
MetaData >> date
     ....
     with a default date if there is none in the metadata

```

this will probably only works when we have metadata so on MicElement we should define another 

```
date
    ^ Date ....1980
```